add ui pack 13.3.1 (#440)
MicroEJ/docs: PlatformDeveloperGuide/uiImageGenerator.rst

5. Copy the generated jar: ``target~/artifacts/imageGeneratorMyPlatform.jar`` in the platform configuration project folder: ``MyPlatform-configuration/dropins/tools/``
6. Rebuild the platform.

> To quickly test a project without rebuilding the Platform or manually exporting the project, add the add the :ref:`Application Option <application_options>` ``ej.imagegenerator.extension.project`` to the absolute path of an Image Generator Extension project (e.g. ``c:\\mycompany\\myimagegeneratorextension``). 

Reviewer: ```suggestion
To quickly test a project without having to rebuild the Platform or manually export the project, add the :ref:`Application Option <application_options>` ``ej.imagegenerator.extension.project`` to be the absolute path of an Image Generator Extension project (e.g. ``ej.imagegenerator.extension.project=c:\\mycompany\\myimagegeneratorextension``). 
```

Reviewer: Same as for the front panel project I would put this in a note and merge the current note+warning

Author: I have moved this advanced notion in a dedicated sub-chapter like Front Panel

> The Image Generator will use the specified Image Generator Extension project before the one included by the Platform.

Reviewer: ```suggestion
The Image Generator will use the specified Image Generator Extension project instead of the one included in the Platform.
```

> .. note:: This feature only works if the Platform has been built with the Image Generator module enabled.

Reviewer: merge with the warning?

Author: I have moved this advanced notion in a dedicated sub-chapter like Front Panel

> .. warning:: This feature is useful for locally testing certain changes in the Image Generator Extension project. The Platform does not contain these changes. The Platform dropins folder must be updated (and the Platform recompiled) after any changes to the Image Generator Extension project. 

Reviewer: "This feature is useful for locally testing certain changes in the Image Generator Extension project. " could be added to the paragraph "To quickly test..."

```suggestion
.. warning:: Use this feature for local tests only. The Platform will not contain the changes: it requires to update the ``dropins`` folder of the Platform configuration project and rebuild the Platform to apply the changes to the Image Generator Extension project.
```

Author: I have moved this advanced notion in a dedicated sub-chapter like Front Panel
